ramls/loan.json

```diff
@@ -253,6 +253,17 @@
         }
       }
     },
+    "feesAndFines": {
+      "description": "Fees and fines associated with loans",
+      "type": "object",
```

Comment:
This object should disallow additional properties from being included (as that is the FOLIO convention)

Reply:
Done